res/invite_template.eml

```diff
@@ -18,7 +18,7 @@ Matrix. To join the conversation, either pick a Matrix client from
 https://matrix.org/docs/projects/try-matrix-now.html or use the single-click
 link below to join via Element (requires Chrome, Firefox, Safari, iOS or Android)
 
-https://app.element.io/#/room/%(room_id_forurl)s?email=%(to_forurl)s&signurl=https%%3A%%2F%%2Fmatrix.org%%2F_matrix%%2Fidentity%%2Fapi%%2Fv1%%2Fsign-ed25519%%3Ftoken%%3D%(token)s%%26private_key%%3D%(ephemeral_private_key)s&room_name=%(room_name_forurl)s&room_avatar_url=%(room_avatar_url_forurl)s&inviter_name=%(sender_display_name_forurl)s&guest_access_token=%(guest_access_token_forurl)s&guest_user_id=%(guest_user_id_forurl)s
+%(web_client_location)/#/room/%(room_id_forurl)s?email=%(to_forurl)s&signurl=https%%3A%%2F%%2Fmatrix.org%%2F_matrix%%2Fidentity%%2Fapi%%2Fv1%%2Fsign-ed25519%%3Ftoken%%3D%(token)s%%26private_key%%3D%(ephemeral_private_key)s&room_name=%(room_name_forurl)s&room_avatar_url=%(room_avatar_url_forurl)s&inviter_name=%(sender_display_name_forurl)s&guest_access_token=%(guest_access_token_forurl)s&guest_user_id=%(guest_user_id_forurl)s
```

Comment:
This is still quite Element-specific. Would this work with Hydrogen for instance? We could be fancy and allow the client to provide a URL filled with named placeholders.

Perhaps we don't care about this for right now though.

Reply:
I'll see if it is reasonable for a homeserver to calculate the other items here. 👍

Reply:
The way that sydent currently works I don't think this is easily doable since a unique key is included in this URL that sydent generates (`ephemeral_private_key`), see the code around:

https://github.com/matrix-org/sydent/blob/c55007f5729cd347a707fa0334ad96afe7a78952/sydent/http/servlets/store_invite_servlet.py#L71-L80

Reply:
I see, thanks for digging. Note that I think we've talked about this key being either unhelpful or broken in Synapse's current implementation: https://github.com/matrix-org/synapse/issues/6036